src/main/java/com/todoist/pojo/Item.kt

```diff
@@ -0,0 +1,58 @@
+package com.todoist.pojo
+
+open class Item<D : Due> @JvmOverloads constructor(
+        id: Long,
+        open var content: String,
+        open var projectId: Long,
+        priority: Int,
+        due: D?,
+        open var sectionId: Long?,
+        open var parentId: Long?,
+        open var childOrder: Int = 0,
+        open var dayOrder: Int = -1,
+        open var isChecked: Boolean = false,
+        open var isCollapsed: Boolean = false,
+        open var assignedByUid: Long?,
+        open var responsibleUid: Long?,
+        labels: Collection<Long>? = null,
+        open var isInHistory: Boolean = false,
+        open var dateAdded: Long,
+        open var dateCompleted: Long? = null,
+        deleted: Boolean = false
+) : TodoistObject(id, deleted) {
+    var labels: Set<Long>? = labels?.toSet().orEmpty()
+        private set
+
+    open var priority: Int = priority
+        /**
+         * Returns the priority within the bounds defined by [.MIN_PRIORITY] and [.MAX_PRIORITY].
```

Comment:
I would say that it makes sense to maintain it like this because the comment is applicable to the (custom) getter only but the AS detail makes a difference. 🤔

Reply:
Yeah and we can also reword it so that it makes sense as a property-level comment and not just a getter or setter comment.